### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by DaxtroDev**
Update the game over message so people know how to play again!

### CODE REVIEW

File: index.html
Comment (DaxtroIGTV) [inline index.html:1]: Great, seems fair

File: README.md
Comment (DaxtroIGTV) [inline README.md:1]: Not sure about the change ask VP

**Review by DaxtroDev — COMMENTED** [review]
So far seems a good Change

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (DaxtroIGTV) [inline index.html:51]:
```suggestion
<h3 hidden>Game</h3>
```

**Pull request merged**